src/content/docs/practice-folder/lorem.mdx

```diff
@@ -9,14 +9,10 @@ Alice was beginning to get very tired of sitting by her sister on the bank, and
 
 So she was considering in her own mind (as well as she could, for the hot day made her feel very sleepy and stupid), whether the pleasure of making a daisy-chain would be worth the trouble of getting up and picking the daisies, when suddenly a White Rabbit with pink eyes ran close by her.
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam lobortis luctus lobortis. Mauris eget volutpat dui. In ultricies molestie ante at tempor. Cras bibendum facilisis venenatis. Aenean augue urna, volutpat ac commodo eu, viverra vitae augue. Nam ullamcorper vehicula lacus, at dignissim velit sollicitudin a. Nullam sed arcu sem. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Nullam interdum mauris interdum lectus faucibus, id ullamcorper purus gravida. Aliquam sollicitudin maximus justo, a semper quam mollis vel. Nulla mattis quam eget varius venenatis. Fusce arcu tellus, faucibus efficitur malesuada in, sollicitudin quis velit. Nunc vitae urna porta, dapibus lectus a, pellentesque tellus. Nunc dolor orci, euismod non vestibulum et, accumsan ac libero. Donec nec porta arcu, ac placerat nisi. Aliquam ornare eros augue, vel convallis odio feugiat sit amet. 
-
 There was nothing so very remarkable in that; nor did Alice think it so very much out of the way to hear the Rabbit say to itself, “Oh dear! Oh dear! I shall be late!” (when she thought it over afterwards, it occurred to her that she ought to have wondered at this, but at the time it all seemed quite natural); but when the Rabbit actually took a watch out of its waistcoat-pocket, and looked at it, and then hurried on, Alice started to her feet, for it flashed across her mind that she had never before seen a rabbit with either a waistcoat-pocket, or a watch to take out of it, and burning with curiosity, she ran across the field after it, and fortunately was just in time to see it pop down a large rabbit-hole under the hedge.
 
 In another moment down went Alice after it, never once considering how in the world she was to get out again.
 
-Donec nec tempor arcu. Maecenas facilisis ultrices finibus. Curabitur tempor quam sit amet lectus venenatis, vitae scelerisque lectus facilisis. Suspendisse facilisis nisi eu diam maximus ullamcorper. Suspendisse a mauris pharetra, laoreet leo in, volutpat leo. Phasellus cursus cursus lectus. Vestibulum aliquet, sapien sit amet dictum dapibus, ipsum nibh tincidunt leo, eget congue nulla turpis nec tortor. Nulla sed sem rhoncus, aliquam lacus ut, placerat elit. Etiam a dolor ultricies, condimentum metus quis, lobortis ipsum. Sed quis metus non tortor tempor pellentesque. Aenean in elit posuere, hendrerit risus a, mollis ipsum. Duis vel elit erat. Praesent est quam, vulputate a lacus in, porta feugiat diam. Praesent euismod eros ac sem molestie gravida. Vivamus vel dictum mi, ut imperdiet quam. Aenean molestie nunc vitae urna cursus dignissim eu nec diam. 
```

Comment:
How did all this junk get in here?

Reply:
Thanks for cleaning it up!

Reply:
Thanks for the feedback! Very constructive.